pkg/volume/iscsi/iscsi_util.go

```diff
@@ -340,8 +340,8 @@ func (util *ISCSIUtil) AttachDisk(b iscsiDiskMounter) (string, error) {
 
 // globalPDPathOperation returns global mount path related operations based on volumeMode.
 // If the volumeMode is 'Filesystem' or not defined, plugin needs to create a dir, persist
-// iscsi configrations, and then format/mount the volume.
-// If the volumeMode is 'Block', plugin creates a dir and persists iscsi configrations.
+// iscsi configurations, and then format/mount the volume.
+// If the volumeMode is 'Block', plugin creates a dir and persists iscsi configurations.
```

Comment:
This has been fixed in https://github.com/kubernetes/kubernetes/pull/59464 - https://github.com/kubernetes/kubernetes/pull/59464/files#diff-50e99d9d6e7992429dd502362f1491af. Closing.

/close